html/modules/custom/docstore/src/Controller/DocumentController.php

```diff
@@ -568,6 +568,9 @@ protected function createDocumentFromParameters(NodeType $node_type, array $para
     // Invalidate cache.
     Cache::invalidateTags(['documents']);
 
+    // Trigger webhook.
+    docstore_notify_webhooks($node_type->id() . ':create', $this->prepareEntityResourceDataForResponse($document, $provider, TRUE));
```

Comment:
I think that should be moved to the `createDocument()` function to be consistent with the `update` and `delete` notifications.

Also that would be triggered for every document in bulk request, which may not be what we want.

Reply:
Good catch, should not be fired for bulk operations.